apps/src/templates/ProfessionalLearningApplyBanner.jsx

```diff
@@ -179,7 +179,7 @@ class ProfessionalLearningApplyBanner extends React.Component {
               >
                 <div>
                   <button type="button" style={this.styles.button}>
-                    Apply Now
+                    Join us
```

Comment:
can this be translated?

Reply:
Professional Learning is only done in en...

Reply:
oh, looking at the code it looked like we were displaying these to everyone, regardless of language

Reply:
It's true we don't do a language check, but `ProfessionalLearningApplyBanner` is used by `YourSchool` which, while partially translated (to my surprise), is only intended for US schools.